ui/app/templates/rules.html

```diff
@@ -90,6 +90,10 @@ <h3 class="panel-title">
                 ng-click="openNewScheduledRuleChangeModal(rule)">
             Schedule an Update
         </button>
+        <button ng-show="rule.scheduled_change === null" class="btn btn-warning btn-xs"
```

Comment:
This button doesn't need the ng-show applied to it. We use this for Updates and Deletes because a Rule may only have a single Scheduled Change associated with it. Scheduling a Duplication doesn't result in a Scheduled Change associated with the Rule, so a user should be able to do this regardless of of whether or not the Rule currently has a change scheduled.

A couple of styling nits too:
* This should be btn-info (we use -info for creation, -warning for modification, and -danger for deletes)
* It should be placed before "Schedule an Update"